documentation/docs/02-runes/02-$state.md

```diff
@@ -85,7 +111,7 @@ class Todo {
 }
 ```
 
-> [!NOTE] The compiler transforms `done` and `text` into `get`/`set` methods on the class prototype referencing private fields.
+Under the hood, the compiler transforms `done` and `text` into `get`/`set` methods on the class prototype referencing private fields. That means the properties are _not_ enumerable.
```

Comment:
The second part sounds like it could be a note
```suggestion
Under the hood, the compiler transforms `done` and `text` into `get`/`set` methods on the class prototype referencing private fields.

 > [!NOTE] Class properties are _not_ enumerable due to the compiler transforms mentioned just above.
```

Reply:
I avoided the note because I felt it disrupts the reading flow for no sufficient reason